.github/workflows/build_test.yml

```diff
@@ -1,50 +1,48 @@
-name: Standard Build and Test
+name: Build & Test PR
 
 on:
   # allows us to run workflows manually
   workflow_dispatch:
-  pull_request:
-    branches:
-      - master
-    paths-ignore:
-      - '.github/workflows/docker_publish.yml'
-
   push:
-    branches:
-      - master
+  pull_request:
     paths-ignore:
+      - 'docker/Dockerfile'
       - '.github/workflows/docker_publish.yml'
 
-
 jobs:
-  BuildTest:
-    runs-on: ubuntu-latest 
+  build-dependency-and-test-img:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ubuntu_versions : [
+          22.04,
+          ]
+        pkg_mgr : [
+          apt,
+          conda,
+        ]
 
-    container:
-      image: cyclus/cyclus-deps
-    
+    name: Installing Dependencies, Building cyclus and running tests
```

Comment:
Maybe I missed it, but it looks like this workflow just builds cyclus, but does it run any tests? The file name implies that t does.

Reply:
The Dockerfile has a `cyclus-test` stage that is being build in this workflow